test/BlockCipher/McryptTest.php

```diff
@@ -12,8 +12,12 @@
 
 class MCryptTest extends AbstractBlockCipherTest
 {
+
```

Comment:
Extra whitespace line. (I feel really pedantic for pointing this one out here, but so far I haven't seen any actual logic errors or vulnerabilities.)